src/cli/configure/processing/ignoreFile.ts

```diff
@@ -1,25 +1,20 @@
-const splitIntoSanitisedLines = (str: string) =>
-  str
-    .trim()
-    .split(/\r?\n/)
-    .map((line) => line.trim());
-
 export const mergeWithIgnoreFile = (templateFile: string) => (
   inputFile?: string,
 ) => {
-  const templateLines = splitIntoSanitisedLines(templateFile);
+  if (typeof inputFile === 'undefined') {
+    return `${templateFile}`;
+  }
 
-  const templateSet = new Set(templateLines.filter((line) => line !== ''));
+  if (inputFile.includes('# managed by skuba')) {
+    const replacedSkubaSection = inputFile.replace(
+      /# managed by skuba[\s\S]*# end managed by skuba/,
+      templateFile,
+    );
 
-  const inputLines = splitIntoSanitisedLines(inputFile ?? '').filter(
-    (line) => !templateSet.has(line),
-  );
+    return `${replacedSkubaSection}`;
+  }
```

Comment:
What do you think about:

```typescript
const replacedFile = inputFile.replace(/.../, templateFile);

if (replacedFile.includes(templateFile)) {
  return replacedFile;
}
```

It's probably overly-paranoid, but it avoids mismatches between the `.replace` pattern and `.includes` string